include/triton/Analysis/Allocation.h

```diff
@@ -18,16 +18,39 @@ namespace mlir {
 namespace triton {
 class AllocationAnalysis;
 
-SmallVector<unsigned>
-getScratchConfigForCvtLayout(triton::gpu::ConvertLayoutOp op, unsigned &inVec,
-                             unsigned &outVec);
-SmallVector<unsigned> getScratchConfigForCvtLayout(RankedTensorType srcType,
-                                                   RankedTensorType dstType,
-                                                   unsigned &inVec,
-                                                   unsigned &outVec);
-SmallVector<unsigned> getRepShapeForCvtLayout(triton::gpu::ConvertLayoutOp op);
-SmallVector<unsigned> getRepShapeForCvtLayout(RankedTensorType srcTy,
-                                              RankedTensorType dstTy);
+struct ScratchConfig {
+  SmallVector<unsigned> repShape;
```

Comment:
What does "rep" stand for?  i.e. could this be called simply "shape"?

Reply:
It's an old name dropped by nvidia. I think it might stand for "repetitive" because the scratch buffer is reused multiple times. Keeping "rep" could make it consistent with the old naming convention in other functions and variables